docs/source/contributing/review_pr_pymc_examples.md

```diff
@@ -0,0 +1,127 @@
+review covers (or skips).
+
+Part of that is also making sure that the PR description links to the relevant issue(s).
+
+(review_pr/initial_pass)=
+## 2. Try to be concise and clear
+Try to be concise and to the point with your reviews.
+**Make sure to leave actionable responses.**
+You'll need less time to review and the PR author will need less time to go over the review.
+Here are some examples about what this does:
+
+* Skim the notebook from top to bottom before starting your review.
+  This will prevent you from spending some time writing about missing content
+  and suggesting how to add it only to find out it is not missing but badly structured.
+* Do not proofread the text until you are sure no large rewritings will be needed.
+  What is the point of reviewing/fixing a typo when the whole paragraph is to be rewritten?
+* Include some rationale about your comments. Contributors come from a wide range
+  of places, fields and personal contexts and English might not be their (nor your) first language.
+  Adding the reasons behind a suggestion goes a long way in making sure the suggestion
+  is clear and makes it more probable that the author doesn't repeat this pattern again.
+
+and doesn't mean:
+
+* Avoiding mentioning good things from the PR or not thanking the PR author.
+* Being cryptic or writing badly
+
+## 3. Review the code and supporting text
+* Check the intended level of the notebook, both when it comes to code and to text.
+  i.e. beginner notebooks should sacrifice performance for the sake of clarity without even needing
+  an explanation, intermediate and advanced ones should not.
+* In all cases, however, remember that this code is written to be read!
+  There is little to gain from obscure one-liners and much to lose, even in
+  intermediate or advanced notebooks. If some code is necessary but not
+  very relevant and you think takes up too much space, hide that cell/input/output
+  under a toggle button.
+* Make sure that the text is relevant, it explains the code blocks when needed and
+  is up to date with the code.
+* Check diffs and output on ReviewNB
+
+## 4. Review the styling and formatting
+* Ignore ReviewNB. ReviewNB renders notebooks incorrectly when there are diffs involving formatting,
+  it can add non-existent HTML tags to the rendered view, mess up links... Moreover, we don't
+  want to render the notebook by itself but as a page on the example gallery.
+* Check the readthedocs preview and use the MyST notebook format to see the raw text diff
+  and comment on it when it comes to formatting.
+
+For the time being (while we rerun and update the docs to new formatting) you should
+not accept any of the following without a very good reason.
+They are very specific and should be part of the PR scope even if the PR author
+didn't intend to originally. Think of this as CI breaking.
+
+* URLs pointing to PyMC/ArviZ/Aesara docs
+* No post directive or no MyST target at the top of the notebook.
```

Comment:
Perhaps re-word this section to avoid double negatives: "you should not accept any of the following" ... "no post directive".  

Something more along the lines of: "please ensure each of the following" ... "post directive and MyST target" ...